docker-builds/db/init/init.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -e
+
+echo ""
+echo "==================="
+echo " Creating role ..."
+echo "==================="
+psql -v ON_ERROR_STOP=1 --username=postgres <<- EOSQL
+CREATE ROLE metasfresh LOGIN ENCRYPTED PASSWORD 'metasfresh' SUPERUSER INHERIT NOCREATEDB NOCREATEROLE;
+EOSQL
+echo "==========="
+echo " ... done!"
+echo "==========="
+
+echo ""
+echo "======================================="
+echo " Creating database and permissions ..."
+echo "======================================="
+psql -v ON_ERROR_STOP=1 --username=postgres <<- EOSQL
+CREATE DATABASE metasfresh WITH OWNER = metasfresh;
+GRANT ALL PRIVILEGES ON DATABASE metasfresh to metasfresh;
+EOSQL
+echo "==========="
+echo " ... done!"
+echo "==========="
+
+echo ""
+echo "==================="
+echo "Restoring pgdump"
+echo "==================="
+pg_restore -Fc --exit-on-error --username metasfresh --dbname metasfresh /metasfresh_latest.pgdump
+echo "=========="
+echo " ...done!"
+echo "=========="
+
+echo ""
+echo "======================================="
+echo " Applying migrations ..."
+echo "======================================="
+find /docker-entrypoint-initdb.d/migrations -type f -printf '%f\n' | sort | while read f; do
```

Comment:
as talked, pls write some quick comments on Y not the migration-tool

Reply:
done